fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/Sequence.java

```diff
@@ -37,7 +38,12 @@ public void apply(Integer priorVersion, IDatabaseAdapter target) {
         if (priorVersion != null && priorVersion > 0 && this.version > priorVersion) {
             throw new UnsupportedOperationException("Upgrading sequences is not supported");
         }
-        apply(target);
+
+        // Only if VERSION1 then we want to apply, else fall through
+        // Re-creating a sequence can have unintended consequences.
+        if (this.version == 1 && priorVersion == 0) {
+            apply(target);
+        }
```

Comment:
good change, I did find in my local message.log, seems apply has been done again and again for all existing derby databases, really nice to see this change, I guess this should be able to fix the issue I see.  and btw, I guess we will always have to change this sequence.java whenever any schema update is made, really want to know how would we support upgrade from any previous version to this version ...